.github/workflows/pypi-release.yaml

```diff
@@ -0,0 +1,96 @@
+name: Build and Upload xarray to PyPI
+on:
+  release:
+    types:
+      - published
```

Comment:
I usually use `created` here, but this probably doesn't make much of a difference since we don't use "draft releases"

Reply:
Not sure if you've seen this[ page](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release) already. There's this note:

> Note: The prereleased type will not trigger for pre-releases published from draft releases, but the published type will trigger. If you want a workflow to run when stable and pre-releases publish, subscribe to published instead of released and prereleased.